ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md

```diff
@@ -386,6 +393,21 @@ defaults <- node_group <- node_group.node <- node
     # IP is derived from the node id.
     mlag_peer_ipv4_pool: < IPv4_network/Mask >
 
+    # Custom structured config applied to MLAG peer link port-channel id.
+    # Added under port_channel_interfaces.<interface> for eos_cli_config_gen.
+    # Overrides the settings on the port-channel interface level.
+    mlag_port_channel_structured_config: < dictionary >
+
+    # Custom structured config applied to MLAG Peer Link (control link) SVI interface id.
+    # Added under vlan_interfaces.<interface> for eos_cli_config_gen.
+    # Overrides the settings on the vlan interface level.
+    mlag_peer_vlan_structured_config: < dictionary >
+
+    # Custom structured config applied to MLAG underlay L3 peering SVI interface id.
+    # Added under vlan_interfaces.<interface> for eos_cli_config_gen.
+    # Overrides the settings on the vlan interface level.
```

Comment:
```suggestion
    # Overrides the settings on the vlan interface level.
    # "uplink_structured_config" is applied after "structured_config", so it can override "structured_config" defined on node-level.
```